pkg/ami/api.go

```diff
@@ -4,27 +4,27 @@ import (
 	"sort"
 	"time"
 
+	"github.com/pkg/errors"
+
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
-	"github.com/pkg/errors"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha4"
 )
 
 const (
-	// DefaultImageFamily defines the default image family for the worker nodes
-	DefaultImageFamily = ImageFamilyAmazonLinux2
-
 	// ImageFamilyAmazonLinux2 represents Amazon Linux 2 family
-	ImageFamilyAmazonLinux2 = "AmazonLinux2" // Owner 602401143452
+	ImageFamilyAmazonLinux2 = api.NodeImageFamilyAmazonLinux2 // Owner 602401143452
```

Comment:
The constants are being relocated to `api` package, so that they can be used there without dragging AWS dependencies from here, as the `api` package is meant to be free of dependencies.